src/frontend/js/types/Joanie.ts

```diff
@@ -19,7 +17,35 @@ export interface PaginatedParameters {
   offset: number;
 }
 
-export interface QueryParameters extends PaginatedParameters {}
+export interface Organization {
+  id: string;
+  code: string;
+  title: string;
+  logo: {
+    filename: string;
+    url: string;
+    height: number;
+    width: number;
```

Comment:
The signature has changed. Take a look here : 
https://github.com/openfun/joanie/blob/ff03672c6a5f606c09c6622cc29de97b678e613e/src/backend/joanie/tests/core/test_api_organizations.py#L91-L107